test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs

```diff
@@ -122,4 +105,93 @@ public void UpdateValueCoreTest()
             Assert.IsTrue(nodeModel.UpdateValueCoreBool(param));
         }
     }
+
+    /// <summary>
+    /// Class containing tests for some of the migration methods of the NodeModel class
+    /// </summary>
+    [TestFixture]
+    public class NodeModelMigrationTests : DynamoModelTestBase
+    {
+        private NodeModelTestingClass nodeModel;
+        private NodeMigrationData migrationDataTest;
+
+        [SetUp]
+        public void Init()
+        {
+            string documentDynPath = Path.Combine(TestDirectory, @"core\Angle.dyn");
+            XmlDocument xmlDoc = new XmlDocument();
+
+            xmlDoc.Load(documentDynPath);
+
+            migrationDataTest = new NodeMigrationData(xmlDoc);
+
+            XmlElement dsFunctionNode = (XmlElement)xmlDoc.SelectSingleNode("//Dynamo.Graph.Nodes.ZeroTouch.DSFunction");
+            migrationDataTest.AppendNode(dsFunctionNode);
+
+            nodeModel = new NodeModelTestingClass();
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void MigrateToDsFunctionNoAssemblyTest()
+        {
+            var result = nodeModel.MigrateToDsFunctionNoAssembly(migrationDataTest, "nickname", "function");
+
+            var assembly = result.MigratedNodes.First().Attributes["assembly"].Value;
+            var nickname = result.MigratedNodes.First().Attributes["nickname"].Value;
+            var function = result.MigratedNodes.First().Attributes["function"].Value;
+
+            Assert.That(assembly, Is.EqualTo(""));
```

Comment:
No strong feeling here but I think our team have been just using `Assert.AreEqual` in this case

Reply:
You are right! I was working on a personal project and used this method so much... I'll change it right away.